src/utils/utils.go

```diff
@@ -230,3 +230,16 @@ func GenerateRequestContext() context.Context {
 func RequestIDLogger(ctx context.Context, log *logrus.Logger) logrus.FieldLogger {
 	return requestid.RequestIDLogger(log, requestid.FromContext(ctx))
 }
+
+func EtcdPatchRequired(openshiftVersion string) (bool, error) {
```

Comment:
please log! How would u know what pass did the code take - it would be very hard to debug

Reply:
The actual [k8s client functions log when they patch and unpatch](https://github.com/openshift/assisted-installer/blob/8a36dac584f7aeec89db4f4df68bd7c6d5c45308/src/k8s_client/k8s_client.go#L150-L170).
I can put an additional log in the non-patch case though if you think that's necessary.

Reply:
if u'll not do patch/unpatch - then there will be no log. It's better to have it explicit. Than u can search in ELK for example

Reply:
Logged in the callers.